app/views/employee/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Employees</h1>
+
+<table>
+	<tr>
+		<th>Employee Name</th>
+	</tr>
+	<tr>
+	<% emp = Employee.all.order(:first_name) %>
+	<% emp.each do |e| %>
```

Comment:
We should use a full variable name `employee`. It's more typing for you at first but it is easier to read and much  faster to comprehend for you and others in the future.